src/averageAges.js

```diff
@@ -1,5 +1,17 @@
 'use strict';
 
+const INITIAL_VALUE = 0;
+const FEMALE_SEX = 'f';
+const MALE_SEX = 'm';
+
+const getAverageAge = people => {
+  return people.reduce((a, b) => a + b, INITIAL_VALUE) / people.length;
+};
```

Comment:
It is good that you understand how to write code in one line, but now it is less clear and functional than it could be.
```suggestion
const getAverageAge = people => {
  const ageAmount = people.reduce((a, b) => a + b, INITIAL_VALUE);
  return (ageAmount / people.length).toFixed(2);
};
```
So the code becomes clearer, it is easier to maintain and we immediately call toFixed for the result of the calculation and do not have to create unnecessary functions below.